src/App.js

```diff
@@ -1,19 +1,109 @@
 import React from 'react';
+import { TodoList } from './components/TodoList';
 import './App.css';
 
 import users from './api/users';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+class App extends React.Component {
+  state = {
+    userName: '',
+    todo: '',
+    todos: [],
+    userSelected: false,
+    todoSelected: false,
+  }
+
+  addTodo = (event) => {
+    event.preventDefault();
+
+    this.setState(state => ({
+      userSelected: !state.userName,
+      todoSelected: !state.todo,
+    }));
+
+    if (!this.state.userName) {
+      return;
+    }
+
+    if (!this.state.todo) {
+      return;
+    }
```

Comment:
Let's simplify your code.

```suggestion
    if (!this.state.userName || !this.state.todo) {
      return;
    }
```